Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:0.12-onbuild
```

Comment:
The doc page says that it's good to go from 0 to Dockerized quickly but that we should not rely on that on the long run. I'm OK with `onbuild` knowing that we might have to change in future if we want to have more control over the setup. But again, by that time this might have been moved to its own official repo.